posts/2024-01-22_myg-jan/README.md

```diff
@@ -0,0 +1,87 @@
+
+### Teritori
+
+Teritori continues the challenging work of developing Gno Project Manager, a web app that allows anyone to create, fund, review, or manage projects fully on-chain. During the last contributors' call, the team gave a demo of the work achieved so far, in particular regarding the escrow system and completing project milestones so contributors can be paid once each one is completed rather than having to wait until the project finalization. 
+
+Gno Project Manager is a complex goal, and the team has run into some issues with edge cases they hadn’t bargained for in the relationships between grantees and funders. The team is looking for feedback and help identifying edge cases, so if you have any in mind, let them know. Teritori is also working on the conflict solver module and improving the social feed on [https://app.teritori.com/feed?network=gno-teritori](https://app.teritori.com/feed?network=gno-teritori), as well as providing more detailed documentation on their work, which they’ll be releasing in the coming weeks.
+
+### Berty
+
+The Berty team has been busy working on GnoSocial backend implementation. The initial feature set has been implemented [here](https://github.com/gnolang/gnosocial/blob/main/realm/public.gno), including posting and replying to messages and reposting threads. You can keep up with Berty’s journey on GnoSocial in [hackerspace issue 51](https://github.com/gnolang/hackerspace/issues/51), which contains many issues and PRs, such as implementing calls, running tests, and fixing bugs. We’re super excited about pushing the limits of scalability with Berty’s decentralized social platform, and we’ll be looking forward to more demos in the coming weeks.
+### Dragos
+Dragos has successfully launched the Flippando game, and you can try it out on the [testnet here](https://gno.flippando.xyz/flip). If you haven’t been following the progress, Flippando is an on-chain memory game that you can play with your choice of styles, such as dice, colors, and hexagrams. Once you successfully complete a matrix, you can mint the end result as an NFT, which can later be assembled into larger, more complex NFTs to create digital artwork. You can find out more about the game, its creator, and the official roadmap on the site. We’ll also release a blog post soon from Dragos sharing his experience porting Flippando from Solidity to Gno, so stay tuned!
+### Varmeta  
+Varmeta’s update was brief this week since the contributor sync call ran over. We look forward to hearing more about the team’s progress in developing the Unity SDK for Gno next time. You can read more about it on Varmeta’s [hackerspace issue 43 here](https://github.com/gnolang/hackerspace/issues/43).
```

Comment:
```suggestion
Varmeta’s update was brief this week since the contributor sync call ran over. We look forward to hearing more about the team’s progress in developing the Unity SDK for Gno next time. You can read more about it on Varmeta’s [hackerspace issue 43](https://github.com/gnolang/hackerspace/issues/43).
```

Consistent use of: "hackerspace issue [number]"